transforms/universal/merge/dpk_merge/transform.py

```diff
@@ -0,0 +1,138 @@
+# (C) Copyright IBM Corp. 2024.
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+from argparse import ArgumentParser, Namespace
+from pathlib import Path
+from typing import Any
+
+import pyarrow as pa
+from data_processing.transform import AbstractTableTransform, TransformConfiguration
+from data_processing.utils import CLIArgumentProvider
+
+
+short_name = "merge"
+cli_prefix = f"{short_name}_"
+input_dirs_key = "input_dirs"
+input_dirs_cli_param = f"{cli_prefix}{input_dirs_key}"
+
+
+class MergeTransform(AbstractTableTransform):
+    """
+    Implements a simple copy of a pyarrow Table.
+    """
+
+    def __init__(self, config: dict[str, Any]):
+        """
+        Initialize based on the dictionary of configuration information.
+        This is generally called with configuration parsed from the CLI arguments defined
+        by the companion runtime, MergeTransformRuntime.  If running inside the RayMutatingDriver,
+        these will be provided by that class with help from the RayMutatingDriver.
+        """
+        # Make sure that the param name corresponds to the name used in apply_input_params method
+        # of MergeTransformConfiguration class
+        super().__init__(config)
+        input_dirs = config.get(input_dirs_cli_param)
+        self.input_dirs = []
+        if input_dirs == None:
+            # For backwards compatibility after we started encouraging the use of cli prefix in the configuration keys
+            input_dirs = config.get(input_dirs_key)
+        if input_dirs is not None:
+            input_dirs = input_dirs.split(",")
+            for input_dir in input_dirs:
+                self.input_dirs.append(Path(input_dir))
```

Comment:
You need to throw exception if input dirs are None

Reply:
done